basic_crud/basic_crud/api.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+import json
+from fastapi import APIRouter, Request
+from fastapi.exceptions import HTTPException
+import reflex as rx
+from .model import Product
+
+
+async def get_product(spec_id: int):
+    with rx.session() as session:
+        spec = session.query(Product).get(spec_id)
+    return spec if spec else HTTPException(status_code=404)
+
+
+async def list_product(req: Request):
+    with rx.session() as session:
+        specs = session.query(Product).all()
+    return specs
+
+
+async def add_product(req: Request):
+    data = json.loads(await req.body())
+    with rx.session() as session:
+        now = datetime.now()
```

Comment:
```suggestion
        now = datetime.now(timezone.utc)
```

since the db column is looking for a timezone aware timestamp, the code should supply one (preferably in utc)